src/store.js

```diff
@@ -75,7 +75,7 @@ export default {
         // Make sure a webserver is started before updating the iframe,
         // to avoid error messages at startup
         const recurrentCheck = setInterval(() => {
-          Vue.http.head(`http:/${container.name}.${state.factory.baseEndpoint}`).then((headResponse) => {
+          Vue.http.head(`http:/${container.name}.${state.factory.baseEndpoint}/error500.html`).then((headResponse) => {
```

Comment:
should ti be in https instead? (or configurable?)

Reply:
Looks like we are unable to provide HTTPS for the shops in any environment, as their domain cannot be linked to the certificate we have (no wildcard)